indexer-app/src/cmr/indexer/data/concepts/service.clj

```diff
@@ -43,4 +52,5 @@
                                  (map #(service-association->service-concept context %)
                                       service-associations))]
     {:has-formats (boolean (some #(has-formats? context %) service-concepts))
+     :has-spatial-subsetting (some #(has-spatial-subsetting? context %) service-concepts)
```

Comment:
@ygliuvt I didn't use `boolean` here, since the predicate function I added above always returns `true` or `false`, never `nil`. However, it looks like the others also will only return `true` or `false`, since they are using logic operators. Do you know why `boolean` is wrapping those here?

Reply:
This is just defensive coding. While `has-formats` always returns true or false, `has-transforms` does not. So I feel safer to just wrap a boolean around the call.

Reply:
@oubiwann Actually, use `boolean` is a must here as `some` returns either true or nil and in this case, we want to convert nil to `false` for indexing. I bet once you add the `boolean` call, most (if not all) of the failed tests should be fixed.